recipes/sdl_image/all/conanfile.py

```diff
@@ -169,7 +170,11 @@ def generate(self):
         cd = CMakeDeps(self)
         cd.generate()
 
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+
     def build(self):
+        self._patch_sources()
```

Comment:
Nowadays we prefer this to to be done in the source method

Reply:
fixed, thanks